package.json

```diff
@@ -43,7 +44,7 @@
     "verify-travis-appveyor": "^3.0.0"
   },
   "scripts": {
-    "install": "prebuild-install || node-gyp rebuild",
+    "install": "node-gyp-build",
```

Comment:
`node-gyp-build` checks if the module can be `require()`'d, if it fails it will recompile.